metpy/interpolate/grid.py

```diff
@@ -333,6 +334,65 @@ def interpolate_to_grid(x, y, z, interp_type='linear', hres=50000,
     return grid_x, grid_y, img.reshape(grid_x.shape)
 
 
+@exporter.export
+def interpolate_to_level(level_var, interp_var, level, **kwargs):
+    r"""Linear interpolation of a variable to a given vertical level from given values.
+
+    This function assumes that highest vertical level (lowest pressure) is zeroth indexself.
```

Comment:
> zeroth indexself

Huh?

Reply:
auto complete in Atom, whoops! just zeroth index.